virtual_ecosystem/models/animals/animal_communities.py

```diff
@@ -215,20 +227,20 @@ def forage_community(self) -> None:
         plant_list = [plant_community]
 
         for consumer_cohort in self.all_animal_cohorts:
-            if (
-                consumer_cohort.individuals == 0
-            ):  # temporary while finalizing die_cohort placements
-                continue
-
+            # Prepare the prey list for the consumer cohort
             prey_list = self.collect_prey(consumer_cohort)
```

Comment:
Should there be a similar function for the plants? Surely not all animals can eat all the plants, right?

Reply:
They can eat all of the plants... for now. The problem of which animals can eat which plants is unfortunately complex and there is no similar first-pass allometric relationship that captures 70% of the variance. Once the plant module has herbivory-facing trait diversity I will be building something parallel out for managing the plants end of things.